DEPLOYMENT.md

```
@@ -0,0 +1,175 @@
+4. **Build for production:**
+   ```bash
+   npm run build
+   ```
+
+## 🌐 Production Deployment
+
+### GitHub Repository Secrets
+
+Add these secrets to your GitHub repository:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)